fix: Calls DeleteExportBucket before checking the status to catch any errors. (#2269)
mongodb/terraform-provider-mongodbatlas: .changelog/2269.txt

```release-note:bug

> resource/mongodbatlas_cloud_backup_snapshot_export_bucket: Calls DeleteExportBucket before checking for a status update. It avoids the delete operation to hang.

Reviewer: ```suggestion
resource/mongodbatlas_cloud_backup_snapshot_export_bucket: Calls DeleteExportBucket before checking for a status update so that the delete operation doesn't hang.
```
